sound/soc/sof/loader.c

```diff
@@ -492,6 +496,8 @@ int snd_sof_load_firmware_raw(struct snd_sof_dev *sdev)
 			fw_filename);
 	}
 
+	plat_data->fw_offset = 0;
+
```

Comment:
This is confusing a bit. Why is this set to 0 here because there's no extended manifest yet?

Reply:
Because there's no extended manifest at all - loading older FW

Reply:
add a comment then to help the reviewer

Reply:
added